interface/forms/CAMOS/content_parser.php

```diff
@@ -23,17 +24,15 @@ function addAppt($days, $time)
 }
 function addVitals($weight, $height, $systolic, $diastolic, $pulse, $temp)
 {
-//This is based on code from /openemr/interface/forms/vitals/C_FormVitals.class.php
-//if it doesn't work, look there for changes.
+//This is based on code from /openemr/interface/forms/vitals/C_FormVitals.class.phpif it doesn't work, look there for changes.
     $_POST['process'] = 'true';
     $_POST['weight'] = $weight;
     $_POST['height'] = $height;
     $_POST['bps'] = $systolic;
     $_POST['bpd'] = $diastolic;
     $_POST['pulse'] = $pulse;
     $_POST['temperature'] = $temp;
-    require("../vitals/C_FormVitals.class.php");
-    $c = new C_FormVitals();
+    $c = new FormVitalsController();
```

Comment:
Can't find this FormVitalsController() classs

Reply:
This should be fixed.  Not sure why PHPStorm didn't refactor this back, but I've changed this.